README.md

```diff
@@ -163,9 +165,12 @@ editing bindings. So, for example:
 * All your standard horizontal motion, selection, insertion, and
   deletion commands work as usual.
 * To delete your current input, just use `C-a C-k`.
-* To go up a directory, use `M-DEL` to delete it in the minibuffer.
+* To edit by word units use `M-DEL` like usual. To go up a directory
+  you can use `C-M-DEL` (bound to `backward-kill-sexp`).
```

Comment:
Would you mind including something like the quote below? I know that some distributions are moving away from enabling this feature, and that the binding is easily rebound within Emacs, but I think it is good to be clear about what can happen. 

A user who accidentally kills their programs by trying this binding might report this as an issue, and I think it would be better to get ahead of such things.

> Be aware that on some Linux distributions, this binding is used to kill the X server, which can cause loss of unsaved work. In that case, `backward-kill-sexp` is also by default bound to `ESC C-DEL`.

Reply:
Good idea, thanks!